cpp/core/CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ macro(build_protobuf)
     PROPERTIES IMPORTED_LOCATION "${PROTOBUF_STATIC_LIB}" INTERFACE_INCLUDE_DIRECTORIES
                "${PROTOBUF_INCLUDE_DIR}")
   add_dependencies(protobuf::libprotobuf protobuf_ep)
+  include_directories(${PROTOBUF_INCLUDE_DIR})
```

Comment:
Can we use `target_include_directories`?
https://cmake.org/cmake/help/latest/command/target_include_directories.html

Reply:
Sure, I will try it.